diff --git a/node_modules/@types/react-email-editor/index.d.ts b/node_modules/@types/react-email-editor/index.d.ts
index 5e13302..67baadd 100644
--- a/node_modules/@types/react-email-editor/index.d.ts
+++ b/node_modules/@types/react-email-editor/index.d.ts
@@ -192,6 +192,7 @@ export default class Component extends ReactComponent<EmailEditorProps> {
     registerCallback(type: "image", callback: FileUploadCallback): void;
     registerCallback(type: "displayCondition", callback: DisplayConditionCallback): void;
     addEventListener(type: string, callback: EventCallback): void;
+    removeEventListener(type: string, callback: EventCallback): void;
     loadBlank(type: object): void;
     loadDesign(design: Design): void;
     saveDesign(callback: SaveDesignCallback): void;
