diff --git a/node_modules/material-ui-dropzone/dist/index.d.ts b/node_modules/material-ui-dropzone/dist/index.d.ts
index 7ddc4c9..a18d42d 100644
--- a/node_modules/material-ui-dropzone/dist/index.d.ts
+++ b/node_modules/material-ui-dropzone/dist/index.d.ts
@@ -1,7 +1,7 @@
-import { ChipProps } from '@material-ui/core/Chip';
-import { DialogProps } from '@material-ui/core/Dialog';
-import { GridProps } from '@material-ui/core/Grid';
-import { SnackbarProps } from '@material-ui/core/Snackbar';
+import { ChipProps } from '@mui/material/Chip';
+import { DialogProps } from '@mui/material/Dialog';
+import { GridProps } from '@mui/material/Grid';
+import { SnackbarProps } from '@mui/material/Snackbar';
 import * as React from 'react';
 import { DropEvent, DropzoneProps } from 'react-dropzone';

diff --git a/node_modules/material-ui-dropzone/dist/index.es.js b/node_modules/material-ui-dropzone/dist/index.es.js
index 259dbd0..fc01d3f 100644
--- a/node_modules/material-ui-dropzone/dist/index.es.js
+++ b/node_modules/material-ui-dropzone/dist/index.es.js
@@ -10,29 +10,29 @@ import _inherits from '@babel/runtime/helpers/inherits';
 import _objectWithoutProperties from '@babel/runtime/helpers/objectWithoutProperties';
 import PropTypes from 'prop-types';
 import { createElement, Fragment, PureComponent } from 'react';
-import Snackbar from '@material-ui/core/Snackbar';
-import Typography from '@material-ui/core/Typography';
-import { withStyles } from '@material-ui/core/styles';
-import AttachFileIcon from '@material-ui/icons/AttachFile';
-import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import Snackbar from '@mui/material/Snackbar';
+import Typography from '@mui/material/Typography';
+import { withStyles } from '@mui/styles';
+import AttachFileIcon from '@mui/icons-material/AttachFile';
+import CloudUploadIcon from '@mui/icons-material/CloudUpload';
 import clsx from 'clsx';
 import Dropzone from 'react-dropzone';
-import Chip from '@material-ui/core/Chip';
-import Fab from '@material-ui/core/Fab';
-import Grid from '@material-ui/core/Grid';
-import DeleteIcon from '@material-ui/icons/Delete';
-import IconButton from '@material-ui/core/IconButton';
-import SnackbarContent from '@material-ui/core/SnackbarContent';
-import CheckCircleIcon from '@material-ui/icons/CheckCircle';
-import CloseIcon from '@material-ui/icons/Close';
-import ErrorIcon from '@material-ui/icons/Error';
-import InfoIcon from '@material-ui/icons/Info';
-import WarningIcon from '@material-ui/icons/Warning';
-import Button from '@material-ui/core/Button';
-import Dialog from '@material-ui/core/Dialog';
-import DialogActions from '@material-ui/core/DialogActions';
-import DialogContent from '@material-ui/core/DialogContent';
-import DialogTitle from '@material-ui/core/DialogTitle';
+import Chip from '@mui/material/Chip';
+import Fab from '@mui/material/Fab';
+import Grid from '@mui/material/Grid';
+import DeleteIcon from '@mui/icons-material/Delete';
+import IconButton from '@mui/material/IconButton';
+import SnackbarContent from '@mui/material/SnackbarContent';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import CloseIcon from '@mui/icons-material/Close';
+import ErrorIcon from '@mui/icons-material/Error';
+import InfoIcon from '@mui/icons-material/Info';
+import WarningIcon from '@mui/icons-material/Warning';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';

 function isImage(file) {
   if (file.type.split('/')[0] === 'image') {
diff --git a/node_modules/material-ui-dropzone/dist/index.js b/node_modules/material-ui-dropzone/dist/index.js
index 144e82b..f603bcc 100644
--- a/node_modules/material-ui-dropzone/dist/index.js
+++ b/node_modules/material-ui-dropzone/dist/index.js
@@ -16,29 +16,29 @@ var _inherits = _interopDefault(require('@babel/runtime/helpers/inherits'));
 var _objectWithoutProperties = _interopDefault(require('@babel/runtime/helpers/objectWithoutProperties'));
 var PropTypes = _interopDefault(require('prop-types'));
 var React = require('react');
-var Snackbar = _interopDefault(require('@material-ui/core/Snackbar'));
-var Typography = _interopDefault(require('@material-ui/core/Typography'));
-var styles$3 = require('@material-ui/core/styles');
-var AttachFileIcon = _interopDefault(require('@material-ui/icons/AttachFile'));
-var CloudUploadIcon = _interopDefault(require('@material-ui/icons/CloudUpload'));
+var Snackbar = _interopDefault(require('@mui/material/Snackbar'));
+var Typography = _interopDefault(require('@mui/material/Typography'));
+var styles$3 = require('@mui/styles');
+var AttachFileIcon = _interopDefault(require('@mui/icons-material/AttachFile'));
+var CloudUploadIcon = _interopDefault(require('@mui/icons-material/CloudUpload'));
 var clsx = _interopDefault(require('clsx'));
 var Dropzone = _interopDefault(require('react-dropzone'));
-var Chip = _interopDefault(require('@material-ui/core/Chip'));
-var Fab = _interopDefault(require('@material-ui/core/Fab'));
-var Grid = _interopDefault(require('@material-ui/core/Grid'));
-var DeleteIcon = _interopDefault(require('@material-ui/icons/Delete'));
-var IconButton = _interopDefault(require('@material-ui/core/IconButton'));
-var SnackbarContent = _interopDefault(require('@material-ui/core/SnackbarContent'));
-var CheckCircleIcon = _interopDefault(require('@material-ui/icons/CheckCircle'));
-var CloseIcon = _interopDefault(require('@material-ui/icons/Close'));
-var ErrorIcon = _interopDefault(require('@material-ui/icons/Error'));
-var InfoIcon = _interopDefault(require('@material-ui/icons/Info'));
-var WarningIcon = _interopDefault(require('@material-ui/icons/Warning'));
-var Button = _interopDefault(require('@material-ui/core/Button'));
-var Dialog = _interopDefault(require('@material-ui/core/Dialog'));
-var DialogActions = _interopDefault(require('@material-ui/core/DialogActions'));
-var DialogContent = _interopDefault(require('@material-ui/core/DialogContent'));
-var DialogTitle = _interopDefault(require('@material-ui/core/DialogTitle'));
+var Chip = _interopDefault(require('@mui/material/Chip'));
+var Fab = _interopDefault(require('@mui/material/Fab'));
+var Grid = _interopDefault(require('@mui/material/Grid'));
+var DeleteIcon = _interopDefault(require('@mui/icons-material/Delete'));
+var IconButton = _interopDefault(require('@mui/material/IconButton'));
+var SnackbarContent = _interopDefault(require('@mui/material/SnackbarContent'));
+var CheckCircleIcon = _interopDefault(require('@mui/icons-material/CheckCircle'));
+var CloseIcon = _interopDefault(require('@mui/icons-material/Close'));
+var ErrorIcon = _interopDefault(require('@mui/icons-material/Error'));
+var InfoIcon = _interopDefault(require('@mui/icons-material/Info'));
+var WarningIcon = _interopDefault(require('@mui/icons-material/Warning'));
+var Button = _interopDefault(require('@mui/material/Button'));
+var Dialog = _interopDefault(require('@mui/material/Dialog'));
+var DialogActions = _interopDefault(require('@mui/material/DialogActions'));
+var DialogContent = _interopDefault(require('@mui/material/DialogContent'));
+var DialogTitle = _interopDefault(require('@mui/material/DialogTitle'));

 function isImage(file) {
   if (file.type.split('/')[0] === 'image') {
