diff --git a/node_modules/react-scripts/config/webpackDevServer.config.js b/node_modules/react-scripts/config/webpackDevServer.config.js
index bf8c98d..8efe394 100644
--- a/node_modules/react-scripts/config/webpackDevServer.config.js
+++ b/node_modules/react-scripts/config/webpackDevServer.config.js
@@ -104,7 +104,8 @@ module.exports = function (proxy, allowedHost) {
     historyApiFallback: {
       // Paths with dots should still use the history fallback.
       // See https://github.com/facebook/create-react-app/issues/387.
-      disableDotRule: true,
+      // Actually, this is more of a hindrance while working with FCM
+      // disableDotRule: true,
       index: paths.publicUrlOrPath,
     },
     public: allowedHost,
