diff --git a/node_modules/react-email-editor/dist/types.d.ts b/node_modules/react-email-editor/dist/types.d.ts
index c060c80..d7ca0e6 100644
--- a/node_modules/react-email-editor/dist/types.d.ts
+++ b/node_modules/react-email-editor/dist/types.d.ts
@@ -34,3 +34,11 @@ declare global {
         __unlayer_lastEditorId: number;
     }
 }
+
+export interface Design {
+    readonly counters?: object | undefined;
+    readonly body: {
+        readonly rows: object[];
+        readonly values?: object | undefined;
+    };
+}
